src/layer/Popup.js

```diff
@@ -189,8 +193,9 @@ export var Popup = DivOverlay.extend({
 		this._tip = DomUtil.create('div', prefix + '-tip', this._tipContainer);
 
 		if (this.options.closeButton) {
-			var closeButton = this._closeButton = DomUtil.create('a', prefix + '-close-button', container);
-			closeButton.href = '#close';
+			var closeButton = this._closeButton = DomUtil.create('button', prefix + '-close-button', container);
+
+			closeButton.setAttribute('aria-label', 'Close');

```

Comment:
This was fixed in [#7794](https://github.com/Leaflet/Leaflet/pull/7794/files#diff-2b5b78f24babc48279b336b4e9d85623e3d1fd37c43fdfe5e7240e736d5fcf7aR194), FWIW.